app/models/organisation.rb

```diff
@@ -8,10 +8,25 @@ class Organisation < ActiveRecord::Base
   has_many :orders
   has_many :users, through: :organisations_users
 
+  validates :name_en, presence: true, uniqueness: true
+  validates :organisation_type_id, presence: true
+
+  before_validation :upcase_name
```

Comment:
If we actually do intend on making all the names uppercase, we probably want to migrate all the existing data as well

Reply:
I think I'd prefer not to upper or lowercase the names in the actual DB as that affects the display version everywhere in the apps also.

When we do searching, we can set it to case-insensitive (ILIKE) and it will be able to match the various forms.

Reply:
done. I have changed it